OmsAgent/watcherutil.py

```diff
@@ -229,14 +217,14 @@ def upload_telemetry(self):
                             self.write_waagent_event(event)
                             self._hutil_log("Successfully processed telemetry status file: "+sf)
 
-                        except Exception as e:
```

Comment:
Does object **e** contain anything useful?

Reply:
in any case we are calling `traceback.format_exc` after which should have the relevant info